### ISSUE TITLE
[feat]: The "Launch UI" button ought to launch either the .local or .onion depending on config

### ISSUE THREAD

**Issue by Rearden21M** [Start9Labs/cryptpad-startos#8]
Right now, the Launch UI will launch the service as .local if you are on your Embassy via LAN, and as .onion if you are connected via Tor. 

When you are connected on your Embassy via LAN and change the config settings to use Tor instead of LAN, one would naturally expect the "Launch UI" button to take you to the .onion address regardless of whether you are on your Embassy via LAN or Tor, since you told the service that you want to use it via Tor. 

In other words, the "Launch UI" ought to launch either the .local or .onion address depending on what is selected in the config. 

Related issue: https://github.com/Start9Labs/embassy-os/issues/1568

**Comment by chrisguida** [comment]
Thanks for the report! We should probably just have the service connect to both Tor and LAN, not sure if it'll be worth it for beta though.

**Comment by chrisguida** [comment]
Blocked by https://github.com/Start9Labs/embassy-os/issues/1568

**Comment by chrisguida** [comment]
On second thought, this is probably better solved by whenever we allow tor and clearnet simultaneously

**Comment by MattDHill** [comment]
Closing out the CryptPad package: it has been retired and is no longer published to the Start9 registry, and this repo is archived. Thanks for the report.
